File: Algoritmos e programação I/5. (14-04) Lista de Exercícios #2/Beecrowd_1079.py
'''
Beecrowd: Problema 1079
13/04/2024
Sophia Swiercoswski
'''

n = int(input())

for testes in range(n):
    a, b, c = map(float, input().split())
    n = (a * 2 + b * 3 + c * 5) / 10
    print("{:.1f}".format(n))

File: Algoritmos e programação I/11. (21-08) Lista de Exercícios #4/Beecrowd_1079.cpp
/*
Beecrowd: Problema 1079
14/08/2024
Sophia Swiercoswski
*/

#include <iostream>
using namespace std;

int main() {
    // Ler número inteiro de testes
    int teste;
    float n1, n2, n3;
    cin >> teste;

    // Calcular média, onde n1 * 2, n2 * 3, n3 * 5 divididos por 10
    for(int i = 1; i <= teste; i++) {
        cin >> n1 >> n2 >> n3;
        printf("%.1f\n", ((n1 * 2) + (n2 * 3) + (n3 * 5)) / 10);
    }

    return 0;
}